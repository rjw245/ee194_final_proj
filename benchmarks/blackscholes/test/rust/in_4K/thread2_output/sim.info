{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread2_output',
             '-n',
             '2',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../rust/bs',
             '2',
             '../../inputs/in_4K.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (840.554216,
            2.676593,
            106772,
            0,
            0,
            0,
            443979,
            2,
            0,
            416,
            11416,
            0,
            0,
            0,
            109707,
            132584),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=2 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/rust/thread2_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../rust/bs 2 ../../inputs/in_4K.txt'],
 't_elapsed': 1293.6943740844727,
 't_start': 1460948560.171128,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 3382353920L}
