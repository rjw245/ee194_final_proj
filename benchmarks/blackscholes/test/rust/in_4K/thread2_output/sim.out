                                     | Core 0     | Core 1    
  Instructions                       |  276502194 |    8390650
  Cycles                             |  167734812 |  167734812
  IPC                                |       1.65 |       0.05
  Time (ns)                          |   63058200 |   63058200
  Idle time (ns)                     |    2520683 |   60497235
  Idle time (%)                      |       4.0% |      95.9%
Branch predictor stats               |            |           
  num correct                        |    9789243 |     303235
  num incorrect                      |     438164 |       6153
  misprediction rate                 |      4.28% |      1.99%
  mpki                               |       1.58 |       0.73
TLB Summary                          |            |           
  I-TLB                              |            |           
    num accesses                     |   56313193 |    1411847
    num misses                       |        211 |         84
    miss rate                        |      0.00% |      0.01%
    mpki                             |       0.00 |       0.01
  D-TLB                              |            |           
    num accesses                     |  184031599 |    4809870
    num misses                       |       4561 |        207
    miss rate                        |      0.00% |      0.00%
    mpki                             |       0.02 |       0.02
  L2 TLB                             |            |           
    num accesses                     |       4772 |        291
    num misses                       |        441 |        216
    miss rate                        |      9.24% |     74.23%
    mpki                             |       0.00 |       0.03
Cache Summary                        |            |           
  Cache L1-I                         |            |           
    num cache accesses               |   56313193 |    1411847
    num cache misses                 |    1290451 |       1083
    miss rate                        |      2.29% |      0.08%
    mpki                             |       4.67 |       0.13
  Cache L1-D                         |            |           
    num cache accesses               |  184032223 |    4810125
    num cache misses                 |      15858 |       3535
    miss rate                        |      0.01% |      0.07%
    mpki                             |       0.06 |       0.42
  Cache L2                           |            |           
    num cache accesses               |    1306460 |       4622
    num cache misses                 |      13239 |       4041
    miss rate                        |      1.01% |     87.43%
    mpki                             |       0.05 |       0.48
  Cache L3                           |            |           
    num cache accesses               |      13401 |       4047
    num cache misses                 |      11309 |        778
    miss rate                        |     84.39% |     19.22%
    mpki                             |       0.04 |       0.09
DRAM summary                         |            |           
  num dram accesses                  |      12080 |          0
  average dram access latency (ns)   |      86.28 |        inf
  average dram queueing delay        |      32.85 |       0.00
  average dram bandwidth utilization |      0.00% |      0.00%
Coherency Traffic                    |            |           
  num loads from dram                |       3262 |        305
  num loads from dram cache          |          0 |          0
  num loads from remote cache        |          0 |          0
